gamemode/cl_autotaunt.lua

```diff
@@ -0,0 +1,21 @@
+
+local autoTauntEnabled = CreateClientConVar("ph_autotaunt_enabled", 1)
+local minTime = CreateClientConVar("ph_autotaunt_min", 30)
+local maxTime = CreateClientConVar("ph_autotaunt_max", 60)
+local time = math.random(minTime:GetInt(), maxTime:GetInt())
+local timeElapsed = 0
+
+timer.Create( "autotaunt", 0.5, 0, function()
+	if LocalPlayer():Team() == 3 && autoTauntEnabled:GetBool() then
```

Comment:
```suggestion
	if IsValid(LocalPlayer()) && LocalPlayer():Team() == 3 && autoTauntEnabled:GetBool() then
```
When changing to this gamemode I would also get an exception since localPlayer is nil. I'm not sure if that broke anything, but this stops the error